src/bindings/build.py

```diff
@@ -34,6 +34,17 @@
     with open(header, "r") as hfile:
         ffi.cdef(hfile.read())
 
+# we need to easily control a few values for Windows builds
+source = []
+if os.getenv("PYNACL_SODIUM_STATIC") is not None:
+    source.append("#define SODIUM_STATIC")
+
+source.append("#include <sodium.h>")
+
+if os.getenv("PYNACL_SODIUM_LIBRARY_NAME") is not None:
+    libraries = [os.getenv("PYNACL_SODIUM_LIBRARY_NAME")]
```

Comment:
If you compile it yourself it won't necessarily be named that, but as I said before, if anyone is compiling it themselves it is perhaps reasonable to state that they need to learn what the expected name is for linking.